modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -107,6 +107,8 @@ endif::openshift-sdn[]
 
 Violating any of these restrictions results in a broken egress firewall for the project, and might cause all external network traffic to be dropped.
 
+An Egress Firewall resource can be created in the kube-node-lease, kube-public, kube-system, openshift and openshift- projects.
+
```

Comment:
Each project name should be in backticks per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Otherwise, looks good!